document/core/Makefile

```
@@ -12,6 +12,7 @@ DOWNLOADDIR   = _download
 NAME          = WebAssembly
 DECISION_URL  = https://github.com/WebAssembly/meetings/blob/main/main/2024/WG-06-12.md
 TAR           = tar
+DEADLINE      = $(shell python3 ../deadline.py)
```

this could more simply use the `date` utility; unfortunately the GNU and BSD date utilities are approximately 0% compatible.

---

```bash
date -d "+30 days" +%Y-%m-%d 2>/dev/null || date -v +30d +%Y-%m-%d
```
…should work — if we want to avoid needing to add the python script

---

Alternatively, shouldn't it work to inline the Python command?
```suggestion
DEADLINE      = $(shell python3 -c "import datetime; print(datetime.date.today() + datetime.timedelta(days=30))")
```
But even better if we can avoid a one-off Python dependency altogether.

---

Yeah, I didn't think of just trying both GNU and BSD arguments; that's simple enough for me.
And yes, I also thought of inlining the Python command, it was just a question of whether it was better to duplicate it in all 3 of the Makefiles or keep it in one place. I don't have a strong opinion on that, other than to note that the Python dependency is hardly one-off; we already have plenty of Python in the W3C build flow, including Bikeshed itself.